ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-release-4.18.yaml

```diff
@@ -238,6 +238,31 @@ tests:
       CLUSTER_AGE_STEP: "300"
       PACKET_OS: rocky_9
     workflow: openshift-e2e-cert-rotation-shutdown
+- as: e2e-metal-assisted-three-node-recovery
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        OPENSHIFT_VERSION=4.18
+        WORKERS_COUNT=1
+        TEST_FUNC=test_ctlplane_scaleup
```

Comment:
3 node is the default for assisted-framework so I think we can entirely remove the part for scaling-up extra workers post-install.
```suggestion
```

Reply:
removed, thanks!